deanslist/deanslist.py

```diff
@@ -83,7 +83,7 @@ def _establish_session(self, api_key, user_agent):
         s.headers.update(headers)
         return s
 
-    def _get(self, relative_url, **kwargs):
+    def _get(self, endpoint, relative_url, **kwargs):
```

Comment:
needing to pass along the endpoint just for the 2 debug messages seems a bit heavy